textdb/textdb-dataflow/src/main/java/com/google/re2j/PublicRE2.java

```diff
@@ -0,0 +1,66 @@
+package com.google.re2j;
+
+/**
+ * PublicRE2 includes flags from RE2 class
+ * comments about flags are copied directly from RE2.java
+ * 
+ * A commonly needed flag is PublicRE2.PERL, which tells RE2 to use Perl syntax
+ * 
+ * @author zuozhi
+ *
+ */
+public class PublicRE2 {
```

Comment:
Yes we need its flags. We don't need all of them (one or two maybe), but I think putting everything here is more explicit